src/containers/Headers.js

```diff
@@ -27,7 +28,11 @@ const handlers = {
   updateSort: props => event => {
     const isAsc = !hasClass('sorted-asc', event.target.className)
     const id = event.target.getAttribute('data-id')
-    props.refreshData(props.name, {...props.filter, sorts: updateFilterSort(id, isAsc, props.filter.sorts)})
+    const updatedFilter = {...props.filter, sorts: updateFilterSort(id, isAsc, props.filter.sorts)}
+    if (updatedFilter && !updatedFilter.page) {
```

Comment:
why is there an if condition for count but not data?

Reply:
needs to be there